docs/kubernetes.adoc

```diff
@@ -397,6 +397,40 @@ public class HelloWorldTest {
 }
 ----
 
+Let's see how can you execute `oc` or `kubectl` commands as a part of your test.
+
+[source, java]
+.OpenshiftAndK8sExample.java
```

Comment:
Can we include already existing test instead of having it written down explicitly here? This will let us avoid updating both the code and documentation while evolving the code of this feature.